fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/perception/errormodels/BoundingBoxOcclusionModifier.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2023 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.fed.application.ambassador.simulation.perception.errormodels;
+
+import org.eclipse.mosaic.fed.application.ambassador.simulation.perception.PerceptionModuleOwner;
+import org.eclipse.mosaic.fed.application.ambassador.simulation.perception.index.objects.SpatialObject;
+import org.eclipse.mosaic.fed.application.ambassador.simulation.perception.index.objects.SpatialObjectBoundingBox;
+import org.eclipse.mosaic.fed.application.ambassador.simulation.perception.index.objects.TrafficLightObject;
+import org.eclipse.mosaic.lib.math.MathUtils;
+import org.eclipse.mosaic.lib.math.Vector3d;
+import org.eclipse.mosaic.lib.math.VectorUtils;
+import org.eclipse.mosaic.lib.spatial.Edge;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BoundingBoxOcclusionModifier implements PerceptionModifier {
```

Comment:
I'm questioning if we really want to call the occlusion implementations "Modifier", since they do not "modify" the perceived objects, but filter them. For me `BoundingBoxOcclusion` could be enough.

Reply:
I will also adjust this for the `SimpleOcclusionMordifier` -> `SimpleOcclusion` and `WallOcclusionModifer` -> `WallOcclusion`.

Reply:
We will need to adjust the documentation accordingly.

Reply:
Also we could rethink naming of the other modifiers:
What do you think about:
* `PositionErrorModifier` -> `PositionError`
* `DistanceModifer` -> ??? any suggestions?